telluric/__init__.py

```diff
@@ -1,4 +1,4 @@
-from telluric.georaster import GeoRaster2
+from telluric.georaster import GeoRaster2, MutableGeoRaster
```

Comment:
it can be a good opportunity to rename GeoRaster2 to GeoRaster

Reply:
that change is too dificult, to many places with the old name